src/main/scala/hmda/model/filing/lar/Applicant.scala

```diff
@@ -0,0 +1,34 @@
+package hmda.model.filing.lar
+
+import hmda.model.filing.PipeDelimited
+import hmda.model.filing.lar.enums._
+
+case class Applicant(
+    ethnicity: Ethnicity,
+    otherHispanicOrLatino: String,
+    ethnicityObserved: EthnicityObservedEnum,
+    race: Race,
+    raceObserved: RaceObservedEnum,
+    otherNativeRace: String,
+    otherAsianRace: String,
+    otherPacificIslanderRace: String,
+    sex: SexEnum,
+    coSex: SexEnum,
+    sexObserved: SexObservedEnum,
+    coSexObserved: SexObservedEnum,
+    age: Int,
+    coAge: Int,
+    income: String,
+    creditScore: Int,
+    creditScoreType: CreditScoreEnum,
+    debtToIncomeRatio: Option[String] = None,
+    otherCreditScoreModel: String
+) extends PipeDelimited {
+  override def toCSV: String = {
+    s"${ethnicity.ethnicity1.code}|${ethnicity.ethnicity2.code}|${ethnicity.ethnicity3.code}|${ethnicity.ethnicity4.code}|" +
+      s"${ethnicity.ethnicity5.code}|$otherHispanicOrLatino|${ethnicityObserved.code}|${race.race1.code}|${race.race2.code}|" +
```

Comment:
Observed ethnicity is in the wrong position.

Also missing observed co-ethnicity.